docs/hacking.md

```diff
@@ -34,7 +34,7 @@ Then you can run the tests by running:
 
 You can alternatively run the tests in a more explicit fashion by passing the `KINTO_PSERVE_EXECUTABLE` environment variable, which can be found in the virtual environment's `bin` directory:
 
-    $ KINTO_PSERVE_EXECUTABLE="/path/to/venv/bin/pserve" npm run test
+    $ KINTO_PSERVE_EXECUTABLE="`pwd`/venv/bin/pserve" npm run test
```

Comment:
This really depends on what's the actual output of `pwd`; users might still be in their `kinto.js` directory, and that wouldn't work.